samples/matlab_experimental/isentropic.m

```diff
@@ -1,5 +1,5 @@
 function isentropic(g)
-    %% Isentropic, adiabatic flow
+    %% Adiabatic, isentropic flow
```

Comment:
I know this is rephrasing the existing text, but isentropic flow is adiabatic by definition, so including both seems redundant. Maybe isentropic nozzle flow or something similar is good.